src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -125,6 +124,11 @@ private static IComponentActivator GetComponentActivatorOrDefault(IServiceProvid
         /// </summary>
         internal bool IsHotReloading { get; private set; }
 
+        /// <summary>
+        /// Gets whether the renderer has been disposed.
+        /// </summary>
+        internal bool Disposed { get; private set; }
+
```

Comment:
This relates to https://github.com/dotnet/aspnetcore/pull/34218/files#r666564951